doc/00-onboarding-guide.md

```diff
@@ -1,30 +1,31 @@
-# Onboarding process for using Azure Cli Codegen (autorest.az)
+# Onboarding Process for Using Azure Cli Code-gen (autorest.az)
 
-## Step 1. Init the process
+Thanks for selecting Azure Cli Code-gen as approach to onboarding Azure Cli for your RP. No matter you want add a new feature to existing PR or onboard a new RP. This document will provide the guideline for your on-boarding process.
 
-Send mail to amecodegen@microsoft.com when your swagger is ready, then we will
+## Step 1. Send request to AME team in Github
+AME team use github issue as a centralized entry point to trigger the process for new RP onboarding or add feature in existing RP. Please go to [Azure-Cli](https://github.com/Azure/azure-cli) to create a new github issue and following the onboarding template first. 
+With the selection result in template, if you select Azure cli Code-gen as the approach, you will be re-direct back to this document later.
+> ** we are improving the process here with Azure CLI team soon 
 
+## Step 2. Quick Validation
+If it is your first time to use Azure Cli code-gen, please discuss with the contact person. He/She will: 
```

Comment:
>the contact person [](start = 72, length = 18)

please mention a contact person will be assigned to help in Step1.

Reply:
Added

---
In reply to: [449503755](https://github.com/Azure/autorest.az/pull/461#discussion_r449503755) [](ancestors = 449503755)